backend/apps/slack/models/message.py

```diff
@@ -53,6 +54,7 @@ def from_slack(
         self.has_replies = message_data.get("reply_count", 0) > 0
         self.slack_message_id = message_data.get("ts", "")
         self.text = message_data.get("text", "")
+        self.message_body = message_data
```

Comment:
It's already a message. This means we will face something like `message.message_body`.
```suggestion
        self.raw_data = message_data
```